File: ios/Sources/CapacitorShakePlugin/CapacitorShake.swift
import Foundation
import Shake

@objc public class CapacitorShake: NSObject {
    @objc public func start(_ apiKey: String) -> Void {
        Shake.configuration.isCrashReportingEnabled = true
        Shake.start(apiKey: apiKey)
    }

    @objc public func show() -> Void {
        DispatchQueue.main.async {
            Shake.show()
        }
    }

    @objc public func registerUser(_ userId: String) -> Void {
        Shake.registerUser(userId: userId)
    }
}

File: android/src/main/java/com/autograb/app/CapacitorShake.java
package com.autograb.app;

import android.app.Activity;

import com.shakebugs.shake.Shake;

public class CapacitorShake {
    public void start(Activity activity, String apiKey) {
        Shake.setCrashReportingEnabled(true);
        Shake.start(activity, apiKey);
        return;
    }

    public void show() {
        Shake.show();
        return;
    }

    public void registerUser(String apiKey) {
        Shake.registerUser(apiKey);
        return;
    }
}
